test/e2e/README.md

```diff
@@ -0,0 +1,54 @@
+# End-to-End Tests
+
+This section contains information about end-to-end (e2e) tests for the VPP agent.
+
+The e2e tests are written using Go. The e2e test suite can be executed for any of the supported VPP versions.
+
+## Run E2E Test Suite
```

Comment:
Could we add a "prerequisites" section here at a e2e subheading or the top-level `##`?  I think the only prereqs are docker and golang, right?